universal-application-tool-0.0.1/app/controllers/admin/ProgramAdminManagementController.java

```diff
@@ -86,8 +87,24 @@ public Result update(Http.Request request, long programId) {
           roleService.makeProgramAdmins(
               programId, ImmutableSet.copyOf(manageAdminForm.getAdminEmails()));
       Result result = redirect(routes.AdminProgramController.index());
-      if (maybeError.isPresent()) {
-        return result.flashing("error", maybeError.get().message());
+
+      if (maybeError.isPresent()){   
+        Optional<Program> program =
+            programRepository.lookupProgram(programId).toCompletableFuture().join();
+        if (program.isEmpty()) {
```

Comment:
nit: empty line separating the multi-line `if` from its peer method contents

Reply:
Done (i think? let me know if you meant something else)